numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2605,9 +2605,9 @@ def add_newdoc(place, name, doc):
 
       >>> a = a = np.full([9,5,7,3], True, dtype=bool)
       >>> c = np.full([9, 5, 4,3], True, dtype=bool)
-      >>> np.dot(a, c).shape
+      >>> np.dot(a, c).shape # doctest: +SKIP
       (9, 5, 7, 9, 5, 4)
-      >>> np.matmul(a, c).shape # n is 5, k is 3, m is 4
```

Comment:
Is there an easy way to keep the original comment too? Maybe move it to the next line?

Reply:
I restored the comment below the original output while retaining the new SKIP directive until fixed by someone later.